e2e/bitrise.yml

```diff
@@ -8,11 +8,8 @@ workflows:
     envs:
     - TEST_APP_URL: https://github.com/bitrise-io/android-multiple-test-results-sample.git
     - BRANCH: maintenance
-    - EXPECTED_APK: mylibrary-debug-androidTest.apk
```

Comment:
This E2E was accidentally passing because of a bug in task creation. The step incorrectly executed additional UI testing and unit testing tasks too, which created this UI test APK in the *app module's build folder* (and not in the nested module's build folder. This is why the step was able to find and export the APK, but this was incorrect.

Turns out the artifact searching logic in this step only handles 1 level of directory depth, so it can't find the nested module's build folder (even without wildcards). I created a follow-up ticket to improve this (STEP-20355), but in the meantime I think the best workaround is to not treat export errors as step failures, allow the step to succeed in this case.